### ISSUE TITLE
Enhancement: Support for Localization of Numbers and Dates

### ISSUE THREAD

**Issue by rodrigoGA** [elevenlabs/elevenlabs-python#253]
I've noticed that elevenlabs translates numbers and dates to audio in English, regardless of the target language. This can compromise the clarity of the generated content for speakers of other languages.

Hotfix Request: It would be beneficial to include a function to transform numbers and dates into text in the desired language before converting to audio. This could serve as a temporary workaround to prevent pronunciation errors and improve overall understanding.

I hope you will consider this proposal for future implementation.

Thank you.

**Comment by stevenzielke** [comment]
I agree that this needs to be a feature but I would recommend SSML. It seems that SSML is not fully enabled in Elevenlabs. Especially not in streaming

Example:
<speak>
  <say-as interpret-as="date" format="yyyymmdd" detail="1">
    2024-04-09
  </say-as>
</speak>

**Comment by Maxamillian-Bonaparte** [comment]
I personally use https://github.com/savoirfairelinux/num2words to do conversion of my text before sending it to the API. It's a good workaround for the moment.

**Comment by AngeloGiacco** [comment]
hey! we are going to be using this page only to track issues with the python sdk but i have logged this as a feature request and will work on getting this set up for you all. i think this will have been solved by our latest models
